src/Controller/TodoController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Todo;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
```

Comment:
Might not be necessary to inherit from AbstractController if no shared helper methods are being used.